dist/app.css

```diff
@@ -226,7 +226,7 @@ a {
     text-decoration: none; }
```

Comment:
Don't worry about commiting changes in `app.css`: there's a task to generate the dist files when the version is bumped.

Reply:
Oh yeah… the release process. I've been using conventional-commits to help with this. Would you agree that this commit is probably of 'fix', so will bump the patch version?

Reply:
Yep, agree